src/common/common.cpp

```diff
@@ -803,4 +804,23 @@ std::optional<std::string> exec_and_read_output(const std::wstring_view command,
 
     CloseHandle(piProcInfo.hProcess);
     return childOutput;
-}
\ No newline at end of file
+}
+
+bool IsAPIContractVxAvailable(uint16_t APIVersion)
+{
+    static bool isAPIContractVxAvailableInitialized = false;
+    static bool isAPIContractVxAvailable = false;
+
+    if (!isAPIContractVxAvailableInitialized)
```

Comment:
use `call_once` instead of using this flag?

Reply:
Same as for the previous comment.